app/controllers/session_helper.rb

```diff
@@ -8,9 +8,7 @@ def session_get(key)
        session[key]
     end
     
-    def session_removeDel(val)
```

Comment:
Move helper class to helper method instead of controller.  It should not be part of controller.